lib/galaxy/tool_util/parser/util.py

```diff
@@ -5,6 +5,7 @@
 
 DEFAULT_METRIC = "mae"
 DEFAULT_EPS = 0.01
+DEFAULT_PIN_LABELS = ""
```

Comment:
An empty string as the default value seems odd, is there a problem with making that None ?

Reply:
Right. Fixed that in 8515eed646a578af96cf7ab3c9ca598dcf0f6ba3.